csm/core/blogic/const.py

```diff
@@ -964,3 +964,9 @@
 
 # CSM usage quotas
 CSM_ACTIVE_USERS_QUOTA = 50
+CSM_ACTIVE_REQUESTS_QUOTA = 100
+MAX_MEM_PER_REQUEST = 5
+
+USAGE_REQUEST_QUOTA = 'usage>request_quota'
```

Comment:
@avrnv Please use same key name in csm config as that of global configuration. That way it is easy to handle that during upgrade. Covered this change for all other keys in my upgrade PR.

Reply:
@Pranali04796 , CSM_USAGE_LIMIT_SERVICES is a destination of an array with limits for all CSM services (currently only agent). Global configuration does not include the quota, so there is no entity with the same meaning as USAGE_REQUEST_QUOTA.